controllers/release/adapter_test.go

```diff
@@ -1241,31 +1238,42 @@ var _ = Describe("Release adapter", Ordered, func() {
 		})
 	})
 
-	When("calling registerAttributionData", func() {
+	When("calling validateAuthor", func() {
 		var adapter *adapter
+		var conditionMsg string
 
 		AfterEach(func() {
 			_ = adapter.client.Delete(ctx, adapter.release)
 		})
 
 		BeforeEach(func() {
 			adapter = createReleaseAndAdapter()
+			adapter.ctx = toolkit.GetMockedContext(ctx, []toolkit.MockData{
+				{
+					ContextKey: loader.ReleasePlanContextKey,
+					Resource:   releasePlan,
+				},
+			})
 		})
 
-		It("returns nil if the release is already attributed", func() {
+		It("returns valid and no error if the release is already attributed", func() {
 			adapter.release.Status.Attribution.Author = "user"
-			err := adapter.registerAttributionData(releasePlan)
-			Expect(err).To(BeNil())
+			valid, err := adapter.validateAuthor()
+			Expect(valid).To(BeTrue())
+			Expect(err).NotTo(HaveOccurred())
 		})
 
-		It("properly sets the Attribution author in the manual release status", func() {
-			adapter.release.Labels = map[string]string{
-				metadata.AuthorLabel: "user",
-			}
-			err := adapter.registerAttributionData(releasePlan)
-			Expect(err).To(BeNil())
-			Expect(adapter.release.Status.Attribution.StandingAuthorization).To(BeFalse())
-			Expect(adapter.release.Status.Attribution.Author).To(Equal("user"))
+		It("should return invalid and no error if the ReleasePlan is not found", func() {
+			adapter.ctx = toolkit.GetMockedContext(ctx, []toolkit.MockData{
+				{
+					ContextKey: loader.ReleasePlanContextKey,
+					Err:        errors.NewNotFound(schema.GroupResource{}, ""),
+				},
+			})
+
+			valid, err := adapter.validateAuthor()
+			Expect(!valid).To(BeTrue())
```

Comment:
Man, `Expect(valid).To(BeFalse())` please :stuck_out_tongue_winking_eye:

Reply:
fixed